manifest.webapp

```diff
@@ -56,6 +56,10 @@
     }
   },
   "permissions": {
+    "terms": {
+      "description": "Required to save accepted applications terms",
+      "type": "io.cozy.terms"
+    },
```

Comment:
Since the Store in installed from the platform, there aren't any translations for the permissions descriptions. I can added some here but it won't be used anywhere for now.